CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+## How to contribute
+
+The purpose of this page is to provide you with the minimum valuable information about how to contribute to the
+project.
+
+### How reach us 🗞
+
+- you can join the slack community [here](https://androiddevs.it/)
+- and follow the open [discussion on the repository](https://github.com/androiddevelopersitalia/aditogether/discussions)
+
+### Set up the project ⚙️
+
+- Have a JDK >= 11 installed (even the bundled one with AS/IJ)
+- Clone the repository
+- Run `./scripts/tuner.sh` to set up the local tools used by this repository
```

Comment:
I would add two more words here, explaining what's the tool for. An idea: 

```suggestion
- Run `./scripts/tuner.sh` to set up the local tools used by this repository: this will help to follow the guidelines and code-style, by running some checks before the commit.
```

Reply:
@4face-studi0 the addition you proposed makes me think that the tuner scripts are running checks before the commit which is not entirely true because:
1. it doesn't run anything before the commit, it installs other scripts. This can maybe misunderstood thinking that someone has to run `tuner.sh` before committing to the project
2. it doesn't install only commit hooks

My proposal:

> - Run `./scripts/tuner.sh` to set up the local tools used by this repository: this script will perform some environmental checks and help to follow our guidelines (e.g. installing Git hooks)